### DISCUSSION TITLE
Math Fontsize in PDF output

### DISCUSSION THREAD

**Discussion by PascalGitz** [quarto-dev/quarto-cli#8085] [Q&A] [answered]
### Description

Hello

I'm currently writing a thesis using Quarto. I would like to reduce the math fontsize in my text, aswell as in $$ $$ environments. The image below shows, that the mathsymbols appear to be too big in the text.

![image](https://github.com/quarto-dev/quarto-cli/assets/115550508/ce47502e-c1f2-43e3-afe3-6f5aabebe471)

this is my quarto.yml file:

```
project:
  type: book

book:
  title: Tragverhalten von Stahlbetontragwerken
  subtitle: Ansätze zur Verformungsberechnung
  author: Pascal Gitz
  date: 2024.01.19
  date-format: full
  chapters:
    - index.qmd
    - 04_Theorie.qmd
    - 06_Versuch_2_A3_Jaeger.ipynb
    - 07_Versuch_SV_14.ipynb
    - 98_summary.qmd
    - 99_references.qmd

bibliography: references.bib

format:
  pdf:
  # Titelblatt und Kurzfassung
    template-partials:
      - title.tex
    include-in-header: 
      - file: styling_nach_hslu.sty
    include-before-body:
      - file: 01_redlichkeitserklaerung.tex
      - file: 02_kurzfassung.tex
  
  # Dokument
    documentclass: scrreprt
    toc-depth: 2
    lang: de
    # keep-tex: true

  # Figuren
    fig-pos: 'H'
    fig-format: svg
  
  # Verlinkungen
    csl: diabetologia.csl
    link-citations: True
    linkcolor: Blue
  
  # Tabellen
    tbl-colwidths: [50,50]
    
  # Code
    execute:
      echo: false
      warning: false
  
  # Font
    mainfont: Palatino Linotype
    fontsize: 12pt
    classoption: egregdoesnotlikesansseriftitles
```

In my styling file,
 ```
 include-in-header: 
      - file: styling_nach_hslu.sty
```
I tried to use with no effect:

`\DeclareMathSizes{3}{3}{3}{3}
`

Does anyone have a suggestion how to set the font size of math symbols?
Thanks for the answers

**Comment by mcanouil** [comment]
Take a look at the `mathfontoptions` option (<https://quarto.org/docs/reference/formats/pdf.html#fonts>).

Also, you can use `keep-tex: true` in your frontmatter and play around the LaTeX code to get the desired results, then put the "answer" back in `include-in-header` for example.
Places dedicated to LaTeX hold the answer your are looking for if the `mathfontoptions` is not.

**Reply by PascalGitz** [reply, marked as answer]
Thank you for the answer. By playing around with the LaTeX code, i got my desired output using the latexmk engine. I assume it has something to do with this snipped in the preamble:

```
\usepackage{amsmath,amssymb}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
```

in short, using `pdf-engine: latexmk` in my quarto.yml solved my problem.
